src/LegacyLinkBlockRepository.php

```diff
@@ -104,6 +105,7 @@ public function createTables()
             "CREATE TABLE IF NOT EXISTS `{$this->db_prefix}link_block_shop` (
     			`id_link_block` int(10) unsigned NOT NULL auto_increment,
     			`id_shop` int(10) unsigned NOT NULL,
+                `position` int(10) unsigned NOT NULL default '0',
```

Comment:
The reason why I hate multiline strings.
Tabs and spaces are mixed :sweat_smile: